### PR TITLE
#82 Feat: Update Front-end #79 Feat: Currency API is written

### PR DESCRIPTION
**Pull request by ufukyilmaz**

### CODE REVIEW

**Comment by irem-ustunboyacioglu** [comment]
I have run your code on localhost, it works fine!!

**Comment by irem-ustunboyacioglu** [comment]
Adding some comments to the code might be helpful for future actions of the project.

File: practice-app/API_DOCUMENTATION.md [hunk 1]
```diff
@@ -23,3 +23,127 @@ Filters by country or/and category and returns news from last 10. If no filter i
...
+                     $("#GBP_ER").text(curr.rates.GBP);
+                     $("#CAD_ER").text(curr.rates.CAD);
+                     $("#SEK_ER").text(curr.rates.SEK);
+                     $("#CHF_ER").text(curr.rates.CHF);
+                 });
+         });
+     </script>
+             </body>
+         </html>}
+
```
Comment (emirhansarac29) [inline practice-app/API_DOCUMENTATION.md]: Returning HTML page is not a good example for REST API function

Comment (ufukyilmaz) [reply practice-app/API_DOCUMENTATION.md]: Got it! I will change it in the following days.

**Pull request merged**